src/eventgrid/azext_eventgrid/commands.py

```diff
@@ -58,6 +138,77 @@ def load_command_table(self, _):
         g.command('delete', 'delete')
         g.custom_command('update', 'cli_domain_update')
 
+    with self.command_group('eventgrid topic private-endpoint-connection', private_endpoint_connections_mgmt_util, client_factory=private_endpoint_connections_factory) as g:
```

Comment:
Based on my knowledge, `private-endpoint-connection` group should also support `approve` and `reject` command (e.g.: `az keyvault private-endpoint-connection -h`, `az storage account private-endpoint-connection -h`).
But as you said that this PR is need to be merged urgently before next Monday, so it's ok for me to approve it, although the functionality maybe insufficient.

Reply:
@ahamad-MS Please take a look at this and other reviewers' suggestion and fix them as possible as you can.

Reply:
Thanks @bim-msft   We are evaluating this in the team and will take as a bug fix next if needed. This should not block this PR for now.